app-emulation/virtualbox/files/virtualbox-7.0.0-fix-compilation-clang.patch

```diff
@@ -0,0 +1,44 @@
+clang does not have syslimits.h, it is gcc specific
```

Comment:
Send upstream if didn't already? (and link here)

Reply:
Compilation with clang is kind of work in progress.  It compiles on my machine with the default USE flags I use, but I didn't check all the flags and I didn't fix virtualbox-modules or virtualbox-guest-additions yet.  When I finish I'll send the patch upstream and update the patchset.